deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 
-sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
+export ALLUXIO_UNDERFS_ADDRESS="gs://${GCS_BUCKET}"
 
-sed -i "/export ALLUXIO_JAVA_OPTS+=\"/ a\
-  -Dfs.gcs.secretAccessKey=${GCS_KEY}" /alluxio/conf/alluxio-env.sh
-
-sed -i "/export ALLUXIO_JAVA_OPTS+=\"/ a\
-  -Dfs.gcs.accessKeyId=${GCS_ID}" /alluxio/conf/alluxio-env.sh
+export ALLUXIO_JAVA_OPTS+="
```

Comment:
for this to take effect, alluxio-env.sh needs to be fixed ... I will send you a fix shortly